native/cocos/physics/physx/joints/PhysXDistance.cpp

```diff
@@ -32,7 +32,8 @@ namespace cc {
 namespace physics {
 
 PhysXDistance::PhysXDistance() : _mPivotA(physx::PxZero),
-                                 _mPivotB(physx::PxZero) {}
+                                 _mPivotB(physx::PxZero),
+                                 PhysXJoint(){}
```

Comment:
I think don't have to invoke PhysXJoint() explicitly.

Reply:
ok